editor/src/components/editor/actions/actions.ts

```diff
@@ -2093,6 +2100,15 @@ export const UPDATE_FNS = {
           derived,
         )
         const parentPath = EP.getCommonParent(orderedActionTargets)
+        const indexInParent = optionalMap(
+          (firstPsthMatchingCommonParent) =>
```

Comment:
`Psth`?

Reply:
and this is now 4 times in the codebase! :D will fix it in a moment

Reply:
fixed